canonical/ubuntu.com
# [WD-7240] remove page for EdgeX IoT and redirect to landing IoT

## Done

- Remove /internet-of-things/edgex page
- Redirect to IoT landing page at /internet-of-things
- Remove EdgeX from navigation items of the IoT landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Navigate to /internet-of-things/edgex and check if it's redirected to /internet-of-things
- Check if EdgeX is not in the list of items in navigation bar of/internet-of-things page

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7240

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
